src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,78 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int LOG_MIN_LEN = 1;
+    public static final int PASS_MIN_LEN = 6;
+    public static final int AGE_MIN = 18;
+    public static final String THE_USER_IS_NULL =
```

Comment:
```suggestion
    private static final String USER_IS_NULL_MESSAGE =
```
Don’t  use articles  in names